src/microbe_stage/Microbe.Interior.cs

```diff
@@ -1449,7 +1468,7 @@ private void HandleDigestion(float delta)
 
             if (engulfable.RequisiteEnzymeToDigest != null)
             {
-                if (!Enzymes.ContainsKey(engulfable.RequisiteEnzymeToDigest))
+                if (CanDigestObject(engulfable) == DigestCheckResult.MissingEnzyme)
```

Comment:
I think a bit of defensive programming would be good here. I'd split this to two levels: the first level gets the digest check result and enters an if when it is not "ok". Then a second level check checks for the exact errors. This ensures that when new error conditions are added this code won't accidentally let those past.

Reply:
I just realized this logic needs to be tweaked. Perhaps switch case is more appropriate?

Reply:
Probably if there's potential to have a bunch of values, I wouldn't really bother with a switch if the enum is ever going to have only 3 or 4 potential values.

Reply:
We just want the benefit of `default` really.